default_warehouse_from_sale_team/i18n/es_MX.po

```diff
@@ -4,14 +4,256 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: Odoo Server 8.0\n"
+"Project-Id-Version: Odoo Server 12.0+e\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2016-08-08 15:50+0000\n"
-"PO-Revision-Date: 2016-08-08 15:50+0000\n"
+"POT-Creation-Date: 2019-05-02 18:26+0000\n"
+"PO-Revision-Date: 2019-05-02 18:26+0000\n"
 "Last-Translator: <>\n"
 "Language-Team: \n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: \n"
 "Plural-Forms: \n"
 
+#. module: default_warehouse_from_sale_team
+#: model:res.groups,name:default_warehouse_from_sale_team.group_manager_default_journal
+msgid "Access to all Journal"
+msgstr "Acceso a todos los diarios"
+
+#. module: default_warehouse_from_sale_team
+#: model:res.groups,name:default_warehouse_from_sale_team.group_manager_default_warehouse_spt
+msgid "Access to all picking types"
+msgstr "Acceso a todos los tipos de albarán"
+
+#. module: default_warehouse_from_sale_team
+#: model:res.groups,name:default_warehouse_from_sale_team.group_manager_default_warehouse_sp
+msgid "Access to all pickings"
+msgstr "Acceso a todos los albaranes"
+
+#. module: default_warehouse_from_sale_team
+#: model:ir.model.fields,help:default_warehouse_from_sale_team.field_crm_team__journal_team_ids
+msgid "Choose the Journals that user with this sale team can see"
+msgstr "Elija las revistas que el usuario con este equipo de venta puede ver"
```

Comment:
wey no mames

Reply:
jajaja, No lo puedo creer jaja

Reply:
Como diste con esto? esto es de hace un año

Reply:
I noticed the reason some features are not working in v11 is that they were fixed/applied only in v12, so I'm back-porting them on https://github.com/Vauxoo/addons-vauxoo/pull/1416